source/renderer/app/containers/wallet/dialogs/wallet-import/StepWalletFileImportContainer.js

```diff
@@ -0,0 +1,61 @@
+// @flow
```

Comment:
@DeeJayElly maybe would be better to set something like `WalletFileImportStepContainer` for a file name as we have on other places. Step as a prefix in file name is a little bit ugly to me. What you think?

Reply:
Agree, changed